### PR TITLE
Feature/trusted curators

### PR DESCRIPTION
**Pull request by creisle**
Breaking Changes
- Move CIViC to its own subparser

New Feature
- add trustedCurators option for CIViC loader

Bug Fixes
- Allow null for disease field in CIViC evidence items
- Use "pending" status for non-accepted evidence items loaded from CIViC

### CODE REVIEW

File: src/civic/README.md [hunk 1]
```diff
@@ -14,3 +14,14 @@ The `variant.js` module handles all parsing/processing of variant records in CIV
 
 In general this loader attempts to update existing records based on their CIViC evidence ID, however when the mapping from CIViC evidence record to GraphKB statement is not 1 to 1 then the
 statement is sometimes recreated and the old version soft-deleted instead since we cannot resolve tracking between the 2 for these cases.
+
+### Trusted Curators
+
+By default this loader only loads accepted (reviewed and approved) evidence items from CIViC. However,
```
Comment (elewis2) [inline src/civic/README.md:20]: the max line width in this section seems to be different from the width in the 'About' section

File: src/civic/index.js [hunk 2]
```diff
@@ -20,9 +20,6 @@ const { downloadVariantRecords, processVariantRecord } = require('./variant');
 
 const ajv = new Ajv();
 
-
-const TRUSTED_CURATOR_ID = 968;
```
Comment (elewis2) [inline src/civic/index.js:24]: was there already one civic curator with trusted status, and this change just allows for any number of trusted curators?

File: src/civic/README.md (hunk 1 above)
Comment (creisle) [reply src/civic/README.md:20]: I had a "soft" max line width of 100 characters but i found that I would have to just re-do the entire paragraph when a word was added so I have stopped using them in md

File: src/civic/index.js (hunk 2 above)
Comment (creisle) [reply src/civic/index.js:24]: yup, but we have now generalised this so external people can pick/choose their trusted curators

File: src/civic/README.md (hunk 1 above)
Comment (creisle) [reply src/civic/README.md]: I've removed all the max-width breaks

**Review by elewis2 — APPROVED** [review]

File: src/civic/index.js [hunk 3]
```diff
@@ -410,15 +416,21 @@ const processEvidenceRecord = async (opt) => {
         evidence: [rid(publication)],
         evidenceLevel: [rid(level)],
         relevance: rid(relevance),
-        reviewStatus: 'not required',
+        reviewStatus: (rawRecord.status === 'accepted'
+            ? 'not required'
+            : 'pending'
```
Comment (cgrisdale) [inline src/civic/index.js:421]: just curious why this change is needed when we are only requesting 'accepted' evidence, other than from trusted curators

Comment (creisle) [reply src/civic/index.js:421]: This is not changing due to the added option and more just because it was previously misleading. Its more accurate to port the submitted not accepted entries in as pending

**Review by cgrisdale — APPROVED** [review]
